code/_core/obj/item/tempering/tempering_magazine.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/tempering/magazine
+	name = "magazine improvement"
+	desc = "You shouldn't be seeing this..."
+	desc_extended = "Report this to burger."
+
+/obj/item/tempering/magazine/bluespace
+	name = "Magazine bluespacer"
+	desc = "Have you tried just putting more bullets in it?"
+	desc_extended = "A tightly controlled bluespace pocket condenser. Fits to expand inside a magazine, and increases capacity 10x for that magazine"
+	icon_state = "quality_melee"
+
+	temper_whitelist = /obj/item/magazine
+
+	value = 50000
+
+/obj/item/tempering/magazine/bluespace/can_temper(var/mob/caller,var/obj/item/magazine/I)
+	if (is_magazine(I) == FALSE)
```

Comment:
``x == FALSE`` should be converted into ``!x`` just in case something goes wrong. In byond, ! means not.